CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [BUGFIX] Ingester: in-order data blocks are no longer labelled es being out-of-order if `out_of_order_blocks_external_label_enabled` is set. #4297
```

Comment:
Not sure if this is needed since the original feature is still unreleased? Maybe add a reference to this PR in the previous changelog entry?

Reply:
Adding PR reference to previous entry is enough.

Reply:
Done